core/templates/dev/head/components/outcome_editor_directive.html

```diff
@@ -25,12 +25,12 @@
             Please give Oppia something useful to say here.
           </span>
         </span>
-        <span ng-if="(!isSelfLoopWithNoFeedback(outcome) && outcome.feedback.length === 0) || (isSelfLoopWithNoFeedback(outcome) && suppressWarnings())"
+        <span ng-if="(!isSelfLoopWithNoFeedback(outcome) && outcome.feedback.getHtml() === '') || (isSelfLoopWithNoFeedback(outcome) && suppressWarnings())"
```

Comment:
.isEmpty(), ditto for the other comparison below.

Reply:
done